app/views/graphql_devise/mailer/confirmation_instructions.html.erb

```diff
@@ -2,4 +2,10 @@
 
 <p><%= t('.confirm_link_msg') %></p>
 
-<p><%= link_to t('.confirm_account_link'), "#{message['schema_url']}?#{confirmation_query(resource_name: @resource.class.to_s, redirect_url: message['redirect-url'], token: @token).to_query}" %></p>
+<p>
+  <% if message['schema_url'].present? %>
```

Comment:
Should this path trigger a deprecation warning?

Reply:
not sure we should do that here. Plus, any value can be passed to message, not sure it's necessary